### PR TITLE
Added Tooltips

### PR DESCRIPTION
**Pull request by Poostaq**
Also cleaned up code a little bit

### CODE REVIEW

File: game/AttributesPage/Draggable.gd [hunk 1]
```diff
@@ -5,6 +5,7 @@ onready var draggable_value = preload("res://AttributesPage/DraggableValue.tscn"
 
 
 func get_drag_data(_pos):
+	print("DRAGGING")
```
Comment (Malynna) [inline game/AttributesPage/Draggable.gd:8]: Delete print

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 2]
```diff
@@ -51,21 +51,17 @@ func _set_image(path):
...
 
 
-func _load_ethnicity(_ethnicity):
-	_set_image(ethnicity["splash_art_path"])
-	ethnicity_name.bbcode_text = "[center]%s[/center]" % ethnicity["ethnicity_name"]
-	ethnicity_description.bbcode_text = "%s" % ethnicity["ethnicity_description"].replace("\n", "\n")
+func _load_ethnicity(ethinicity):
+	_set_image(ethinicity["splash_art_path"])
+	ethnicity_name.bbcode_text = "[center]%s[/center]" % ethinicity["ethnicity_name"]
+	ethnicity_description.bbcode_text = "%s" % ethinicity["ethnicity_description"]
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:57]: In my opinion: Ethnicity not ethinicity

**Review by Malynna — CHANGES_REQUESTED** [review]
Fix 2 little bugs and go ahead

File: game/AttributesPage/Draggable.gd (hunk 1 above)
Comment (Poostaq) [reply game/AttributesPage/Draggable.gd]: Fixed

**Pull request merged**